Cargo.toml

```
@@ -121,12 +121,13 @@ pre-release-replacements = [
 ]
 
 [features]
-default = ["toml", "json", "yaml", "ini", "ron", "json5", "convert-case", "async"]
+default = ["toml", "json", "yaml", "ini", "ron", "json5", "dotenv", "convert-case", "async"]
 json = ["serde_json"]
 yaml = ["yaml-rust2"]
 ini = ["rust-ini"]
 json5 = ["json5_rs", "dep:serde-untagged"]
 corn = ["dep:corn"]
+dotenv = ["dotenvy"]
```

```suggestion
dotenv = ["dep:dotenvy"]
```
Otherwise, we'll expose a `dotenvy` feature flag